From 8bb78ab1cef5bc61a8b021a74f7021e09fd16d4b Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Mon, 10 Nov 2025 23:10:31 +0100
Subject: [PATCH 08/32] vdo: add missing synchronization

After wiping, formating and deactivating vdopool volume,
we should wait on udev before we continue and activate
same volume again with different UUID.

(cherry picked from commit 6f9c241c81ecbe521d1554d2d603b8e99f88b9d9)
---
 lib/metadata/vdo_manip.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/metadata/vdo_manip.c b/lib/metadata/vdo_manip.c
index 735323ce4..13344e423 100644
--- a/lib/metadata/vdo_manip.c
+++ b/lib/metadata/vdo_manip.c
@@ -429,6 +429,8 @@ int convert_vdo_pool_lv(struct logical_volume *data_lv,
 		return 0;
 	}
 
+	sync_local_dev_names(cmd);
+
 	vdo_logical_size -= 2 * vdo_pool_header_size;
 
 	if (vdo_logical_size < extent_size) {
-- 
2.52.0

